release-notes/releases-index.json

```diff
@@ -0,0 +1,44 @@
+{
+    "core-releases-index": [
+        {
+            "channel-version": "2.1",
+            "latest": "2.1.4",
+            "release-date":"2018-09-11",
+            "lts": true,
+            "eol": false,
```

Comment:
Instead of "eol" as a property, you might go with: "support-phase": "full-support" | "maintenance" | "eol"

Reply:
Was thinking the same thing after our 'maintenance' exchange. Change will be in the next commit.